crates/phactory/src/prpc_service.rs

```diff
@@ -1057,8 +1059,9 @@ impl<Platform: pal::Platform + Serialize + DeserializeOwned> PhactoryApi for Rpc
         let my_identity_key = system.identity_key.clone();
 
         // 1. verify RA report
+        // this also ensure the message integrity
         let challenge_handler = request.decode_challenge_handler().map_err(from_display)?;
-        let now_ms = system.now_ms;
+        let block_ms = system.now_ms / 1000;
```

Comment:
Why do we divide ms by 1000 here?

Reply:
Naming problem, will rename this to `block_sec`